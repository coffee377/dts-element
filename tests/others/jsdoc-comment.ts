import * as dts from '../index';

it('should return correctly with text', () => {
  expect(dts.emit(
    dts.create_jsdoc_comment({
      text: 'test 1\n\ntest 2',
    }),
  )).toMatchSnapshot();
});
