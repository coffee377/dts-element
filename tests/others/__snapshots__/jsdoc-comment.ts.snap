// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return correctly with text 1`] = `
"/**
 * test 1
 *
 * test 2
 */"
`;
